tagsync/src/main/java/org/apache/ranger/tagsync/process/TagSyncConfig.java

```diff
@@ -120,7 +121,7 @@ public class TagSyncConfig extends Configuration {
 	private static final int     DEFAULT_TAGSYNC_SINK_MAX_BATCH_SIZE = 1;
 	private static final String  TAGSYNC_SINK_MAX_BATCH_SIZE_PROP    = "ranger.tagsync.dest.ranger.max.batch.size";
 
-
+	private static final String TAGSYNC_ATLAS_REST_SOURCE_REQUESTED_ENTITIES_LIMIT_MAX  = "ranger.tagsync.source.atlasrest.requested.entities.limit.max";
```

Comment:
I suggest to rename the property to: ranger.tagsync.source.atlasrest.entities.batch.size

Reply:
Thanks for your help